projects/demo-playwright/tests/kit/input/input.spec.ts

```diff
@@ -0,0 +1,132 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe('Input', () => {
+    test("has custom content (text) + cleaner + hint which don't overlapping each others", async ({
+        page,
+    }) => {
+        await tuiGoto(
+            page,
+            '/components/input/API?tuiMode=null&tuiTextfieldCustomContent=<span>LongTextContent<%2Fspan>&tuiHintContent=Some%20content',
+        );
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '01-input.png',
```

Comment:
Why not `01-custom-text-content-cleaner-hint.png` (as it was in cypress tests)?
(the same question for screenshot name)

Reply:
fixed